pkg/services/oauth/oauth.go

```diff
@@ -84,6 +144,78 @@ func (s *service) CreateAccessToken(request services.CreateAccessTokenRequest) (
 	return &services.AccessTokenResult{AccessToken: accessToken}, nil
 }
 
+// check the actor against the registry, according to RFC003 §5.2.1.3
+// we do this by getting the validation chain for the certificate in the x5c header and check the vendorID SAN from the root
+// with the vendorId of the actor
+func (s *service) validateIssuer(jwtBearerToken *services.NutsJwtBearerToken) error {
+	validationTime := time.Unix(jwtBearerToken.IssuedAt, 0)
+
+	actorPartyID, err := core.ParsePartyID(jwtBearerToken.Issuer)
```

Comment:
do we have enough logging (e.g. of issuer PartyIDs which proved to be invalid) to analyse production issues? Or is the issuer mentioned in the returned error?